Mathlib/MeasureTheory/Integral/IntervalIntegral.lean

```diff
@@ -592,6 +599,14 @@ nonrec theorem integral_ofReal {a b : ℝ} {μ : Measure ℝ} {f : ℝ → ℝ}
     (∫ x in a..b, (f x : ℂ) ∂μ) = ↑(∫ x in a..b, f x ∂μ) :=
   RCLike.intervalIntegral_ofReal
 
+theorem intervalIntegral_re {μ : Measure ℝ} {a b : ℝ} {f : ℝ → ℂ} (hab: a ≤ b)
+    (hf: IntervalIntegrable f μ a b) :
+    (∫ x in (a)..b, (f x).re ∂μ) = (∫ x in (a)..b, f x ∂μ).re := by
+  repeat rw [intervalIntegral.integral_of_le hab]
+  apply setIntegral_re
+  rw [← intervalIntegrable_iff_integrableOn_Ioc_of_le hab]
+  exact hf
```

Comment:
In statement `hf:` -> `hf :`. Again, I suspect the statement is true without `hab`, and for an `RCLike` field rather than just `ℂ`. The proof can be shortened to
```
  simpa only [integral_of_le hab] using 
    setIntegral_re <| (intervalIntegrable_iff_integrableOn_Ioc_of_le hab).mp hf
```
(note that you don't need the fully qualified name `intervalIntegral.integral_of_le`, just `integral_of_le` is enough, because you're already inside the `intervalIntegral` namespace).

Do we want a version of this lemma for `im` as well?